composer/algorithms/low_precision_layernorm/low_precision_layernorm.py

```diff
@@ -99,7 +99,7 @@ def forward(self, x):
         module_device = x.device
         downcast_x = _cast_if_autocast_enabled(x)
         downcast_weight = _cast_if_autocast_enabled(self.weight)
```

Comment:
```suggestion
        downcast_weight = _cast_if_autocast_enabled(self.weight) if self.weight is not None else self.weight
```
needed if `elementwise_affine=False`